crates/bevy_ecs/src/label.rs

```diff
@@ -125,7 +125,7 @@ macro_rules! define_label {
             /// Feeds this value into the given [`Hasher`].
             fn dyn_hash(&self, state: &mut dyn ::std::hash::Hasher);
 
-            /// Returns an [`Interned`](bevy_utils::intern::Interned) value corresponding to `self`.
+            /// Returns an [`Interned`](bevy_ecs::intern::Interned) value corresponding to `self`.
```

Comment:
Changing this to `[`Interned`](Interned)` fixes one of the issues in the docs CI job but there seems to be another issue that I couldn't figure out yet. I'd appreciate some guidance with it.

Reply:
```suggestion
            /// Returns an [`Interned`](crate::intern::Interned) value corresponding to `self`.
```

`crate` is self-referential when inside the target crate.

Reply:
Thanks for applying that change. I'm 99% positive that I tried that and it error-ed out locally but I might be wrong. Let's see if that fixes the CI issue.